portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -64,8 +82,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       return;
     }
 
-    comments.add(comment);
-    jsonObject.put("comments", comments);
+    Entity entity = new Entity("Comment");
+    entity.setProperty("text", comment);
+    entity.setProperty("timestamp", System.currentTimeMillis());
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(entity);
+
+    jsonObject.put("success", "Successfully created comment.");
```

Comment:
consider something like `{"status": "success", "message": "Successfully created comment."}` which would allow you to return errors like: `{"status": "error", "message": "Bad request."}`. Feel free to do this in a follow-up PR.